camera-core/src/main/java/com/stripe/android/camera/framework/util/Layout.kt

```diff
@@ -25,7 +25,7 @@ import kotlin.math.roundToInt
  * ```
  */
 @CheckResult
-internal fun maxAspectRatioInSize(area: Size, aspectRatio: Float): Size {
+fun maxAspectRatioInSize(area: Size, aspectRatio: Float): Size {
```

Comment:
should these all be restricted to the library group?

Reply:
Sounds good, just added the annotation to the public `fun`s